sale_force_invoiced/model/sale_order.py

```diff
@@ -13,12 +13,13 @@ class SaleOrder(models.Model):
         "quantities pending to invoice.",
         readonly=True,
         states={"done": [("readonly", False)], "sale": [("readonly", False)]},
+        tracking=20,
         copy=False,
     )
 
     @api.depends("force_invoiced")
-    def _get_invoice_status(self):
-        res = super(SaleOrder, self)._get_invoice_status()
+    def _compute_invoice_status(self):
+        res = super()._compute_invoice_status()
         for order in self.filtered(
```

Comment:
@alexis-via  You can improve:

`self.filtered(lambda so: so.force_invoiced and so.state in ("sale", "done")).update({"invoice_status": "invoiced"})`

Reply:
@alexis-via can you add it?

Reply:
Everything looks fine @alexis-via. Can you add this change?

Thanks!